web3_proxy/src/config.rs

```diff
@@ -94,6 +94,14 @@ pub struct AppConfig {
     /// None = allow all requests
     pub default_user_max_requests_per_period: Option<u64>,
 
+    /// Default ERC address for out deposit contract
+    pub deposit_contract: String,
```

Comment:
These should be optional. Not everyone who runs the proxy will run it with payments. Also, we might be able to use ether's Address type here